app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/SuperFragment.kt

```diff
@@ -0,0 +1,18 @@
+package edu.uc.kovaciad.slicetracker.ui.main
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProviders
+
+open class SuperFragment : Fragment() {
+    internal lateinit var viewModel: MainViewModel
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+        activity.let {
```

Comment:
A nice trick in Kotlin: change this to activity?.let {}.  That way, the logic within the {} will only execute if activity is not null.